OpenMEEG/libs/OpenMEEG/include/geometry_io.h

```diff
@@ -238,18 +238,16 @@ namespace OpenMEEG {
         // Add the array of Indices to polydata points
         polydata->GetPointData()->AddArray(point_indices);
         // Add optional potentials and currents
-        if (data.nlin() != 0)
+        if (data.nlin() != 0) {
             for (unsigned j = 0; j < data.ncol(); ++j) {
                 polydata->GetPointData()->AddArray(potentials[j]);
                 polydata->GetCellData()->AddArray(currents[j]);
             }
+        }
```

Comment:
don't worry I won't change that! I'm following the coding_rules defined here ./OpenMEEG/coding_guidelines.txt